### ISSUE TITLE
`istioctl ztunnel-config certificates` (and other `ztunnel-config` subcommands) randomly picking an instance is confusing

### ISSUE THREAD

**Issue by bleggett** [istio/istio#53335]
**Describe the feature request**

Right now if you run a `ztunnel-config` istioctl subcommand with no arguments on a cluster, it will _randomly_ pick a ztunnel pod to show you results for, e.g. if you have a 3 node cluster and thus 3 ztunnels:

- `ztunnel-bing` 
- `ztunnel-bang`
- `ztunnel-boom`

`istioctl ztunnel-config certificates` will silently choose `ztunnel-bing` and only show you workload certificates for `ztunnel-bing`.

This is both confusing and not particularly useful UX if you have more than 2 or 3 nodes - if you have 10 nodes (thus 10 ztunnels) and 200 workloads, running `ztunnel-config certificates` with no arguments gives you an effectively useless non-error result: 

- if you do not specify a specific ztunnel instance, you get a result from a random ztunnel (why would I ever usefully need workload certs from _one_ random node out of 10)?
- `istioctl` does not tell you which one it picked, you have to guess :D. With many nodes, this feels like capricious and not-useful behavior.
- `istioctl` does not give you a way to aggregate results from all ztunnels AFAICT, meaning you have to run it 10x: `istioctl ztunnel-config certificates ztunnel-1....istioctl ztunnel-config certificates ztunnel-100`. Naturally you can usually do what you want with bash nonsense and `jq` and such - but that's not exactly ideal.

**Describe alternatives you've considered**

Option A:

`istioctl ztunnel-config` subcommands should return an *error* if you do not provide a specific ztunnel instance to query.

Option B: 

`istioctl ztunnel-config` subcommands should *echo* the ztunnel instance they chose in the output, if you do not provide a specific ztunnel instance to query.

Option C (one that seems most intuitively obvious to me):

`istioctl ztunnel-config` subcommands should aggregate results from *all accessible* ztunnels on *all nodes* in the output, if you do not provide a specific ztunnel instance to query.

Either way, the current behavior is less useful than any of these alternatives - so we should pick one.

**Affected product area (please put an X in all that apply)**

[x] Ambient
[ ] Docs
[ ] Dual Stack
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[x] User Experience
[ ] Developer Infrastructure

**Affected features (please put an X in all that apply)**

[ ] Multi Cluster
[ ] Virtual Machine
[ ] Multi Control Plane

**Additional context**

**Comment by howardjohn** [comment]
I agree with most of this but not that `certificates` is the primary one that has different results per-ztunnel. Services/workloads/policies _should_ be the same for everything so its fairly nice to let it just pick one for most debugging

**Comment by bleggett** [comment]
> I agree with most of this but not that `certificates` is the primary one that has different results per-ztunnel. Services/workloads/policies _should_ be the same for everything so its fairly nice to let it just pick one for most debugging

Yeah - that's fair. `log` and `certificates` are probably the main ones that need either 

- a global specifier
- an instance specifier

and shouldn't just pick a random instance if neither specified.

**Comment by howardjohn** [comment]
I believe `log` already hits all of them
